README.rst

```diff
@@ -151,6 +151,43 @@ For example, if you wanted to know what code paths query the table
 The performance record herer would include a standard Python traceback attached
 to each SQL query containing "my_table".
 
+
+``capture_operation``, if not ``None``, should be a function that takes one
+argument, the given DB or cache operation, and returns a ``bool`` indiciating if
```

Comment:
```suggestion
argument, the given DB or cache operation, and returns a ``bool`` indicating if
```

Reply:
4b1e1e4, #353 fixes another case of this already in the README.